firebase_test.go

```diff
@@ -66,7 +66,7 @@ func TestNewWithProvidedHttpClient(t *testing.T) {
 			return c, err
 		},
 	}
-	var client = http.Client{Transport: tr};
+	var client = http.Client{Transport: tr}
```

Comment:
Good catch. I thought I had travis setup to fail if something failed `gofmt`. I'll have to adjust the `.travis.yml` file.